CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ### NEXT
 
-* Github workflow: Remove deprecated Ubuntu version 18.04 ([PR #1033](https://github.com/versatica/mediasoup/pull/1033)).
+* GitHub CI: Remove deprecated Ubuntu version 18.04 ([PR #1033](https://github.com/versatica/mediasoup/pull/1033)).
+* GitHub CI: Cancel in progress workflows ([PR #1034](https://github.com/versatica/mediasoup/pull/1034)).
```

Comment:
Not sure why users of mediasoup would want to see and care about this. Probably shouldn't be in changelog, if they care about everything that happened in the repo, they can see the list of merged things directly.

Reply:
I definitely agree and will remove these not useful entries from Changelog.